.github/issue-labeler.yml

```diff
@@ -0,0 +1,90 @@
+# archived/webui:
+# backport/1.6:
+# backport/1.7:
+# backport/1.8:
+# beta/1.8:
+# blocks-release:
+# crash:
+# do-not-merge:
+# help-wanted:
+# needs-discussion:
+# needs-investigation:
+# post-beta:
+# pr/dependencies:
+# pr/needs-rebase:
+# pr/needs-tests:
+theme/acls:
+    - '(acl|ACL)'
+# theme/agent-cache:
+# theme/api:

```

Comment:
Should we capture anything with api/API in here already?

Reply:
Unfortunately, for issues, I don't believe so. If someone even says "I used the HTTP API to register the service" but the issue isn't with the API - it becomes an API label. I think that this label works better with PRs than issues, but I captured it as a comment because I wanted to represent it.

I could see this being useful for tracking issues with the HTTPS endpoint and configuration though

Reply:
Jsolauslka PR  endpoint should b capturing with API

Reply:
Hi @Juansbulbisful - Thanks for commenting! There are times where we want to track issues with the API that aren't always PR'd. It'd be wonderful if the labelers out there could do both, but for now, I'm fine with this being a manual one. Periodically, there is a review of labels and their utility, and API will probably get reviewed on the next round.